deploy-satellite-update-playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
There is no need to have the redundant "deploy" or "playbook" in the file title. Can you call this something that highlights that its for minor versions only? i.e `update-satellite-minor-release`